test/selenium/lib/foxboxHelperSelenium.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
This is almost the same No-Jun created for integration. Wanted to try if changing the [path ](https://github.com/isabelrios/foxbox/blob/ui_app_test/test/selenium/lib/foxboxHelperSelenium.js#L24)would fix the test not passing and yeah, that the case.
Then I thought, although there is duplicated code, that having this for UI tests would be helpful, we need kill and start the foxbox(but not with all the services as integration tests do) 
This could be a first approach and then refactor to have one Helper for each if possible?